appengine-java8/springboot-helloworld/pom.xml

```diff
@@ -93,9 +93,16 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>2.4.1</version>
         <configuration>
-          <!-- can be set w/ -DprojectId=myProjectId on command line -->
+          <!--
+            For the details of the parameters, see
+            https://cloud.google.com/appengine/docs/standard/java/maven-reference
+          -->
+          <!-- can be set w/ -Dapp.deploy.version=myProjectId on command line -->
           <projectId>myProjectId</projectId>
-          <!-- set the GAE version or use "GCLOUD_CONFIG" for an autogenerated GAE version -->
+          <!--
+            Your application's version in GAE. To use an autogenerated version, use
+            "GCLOUD_CONFIG" or leave it unspecified.
```

Comment:
My memo. The reference says we can leave it unspecified to get autogenerated version.

https://cloud.google.com/appengine/docs/standard/java11/maven-reference